routes/api.js

```diff
@@ -1,44 +1,34 @@
 'use strict';
 
-var apiConfig = require('../server/config').api;
-var COCKTAIL_PATH = apiConfig.base + apiConfig.cocktailPath;
-var dbOperations = require('../server/db-operations');
+const apiConfig = require('../server/config').api;
+const COCKTAIL_PATH = `${apiConfig.base}${apiConfig.cocktailPath}`;
+const dbOperations = require('../server/db-operations');
 
-module.exports = function (app) {
+module.exports = (app) => {
 
-  app.post(COCKTAIL_PATH, function (req, res) {
+  app.post(COCKTAIL_PATH, (req, res) => {
     dbOperations.createCocktail(req.body)
-      .then(function (dbRes) {
-        return res.status(200).json(dbRes);
-      });
+      .then((dbRes) => res.status(200).json(dbRes));
```

Comment:
Here's an ultra-concise arrow function: when the function body is just one line, you can leave out the braces around it and the `return`.

That is, this is the equivalent of
```
.then((dbRes) => {
  return res.status(200).json(dbRes)
});
```
... which is, of course, the equivalent of
```
.then(function (dbRes) => {
  return res.status(200).json(dbRes)
});
```